api/mapping/templates/mapping/dataset_list.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="{% url 'home' %}">Home</a></li>
+        <li class="breadcrumb-item"><a href="{% url 'datasets' %}">Datasets</a></li>
+    </ol>
+</nav>
+
+<div>
+    <div id='root'></div>
+    <script type="text/javascript" >
+        var isAdmin = "{{is_admin}}" === "True" ? true : false
+    </script>
```

Comment:
I don't think you don't need this since you're not passing `is_admin` from the `dataset_list_page` view.